### PR TITLE
Clean up dependencies in ASMDEFs and manifests

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Clean up some dependencies that are not necessary.

### CODE REVIEW

File: com.microsoft.mrtk.core/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "unity": "2020.3",
   "dependencies": {
     "com.unity.xr.interaction.toolkit": "2.0.2",
-    "com.unity.xr.management": "4.2.1"
+    "com.unity.xr.management": "4.2.1",
+    "com.unity.xr.core-utils": "2.0.0"
```
Comment (keveleigh) [inline com.microsoft.mrtk.core/package.json:20]: Is this already explicitly needed? Or something that's coming?

Comment (keveleigh) [reply com.microsoft.mrtk.core/package.json:20]: Oh, I see it in the asmdef. Nevermind!

File: com.microsoft.mrtk.input/MRTK.Input.asmdef [hunk 2]
```diff
@@ -6,8 +6,7 @@
         "Microsoft.MixedReality.Toolkit.Core",
         "Unity.InputSystem",
         "Unity.XR.CoreUtils",
```
Comment (keveleigh) [inline com.microsoft.mrtk.input/MRTK.Input.asmdef:8]: Should we add `"com.unity.xr.core-utils": "2.0.0"` to the Input package.json as well?

Comment (MaxWang-MS) [reply com.microsoft.mrtk.input/MRTK.Input.asmdef:8]: Added!

**Review by keveleigh — APPROVED** [review]

**Pull request merged**